src/Databricks/docs/New-AzDatabricksVNetPeering.md

```diff
@@ -0,0 +1,315 @@
+---
+external help file:
+Module Name: Az.Databricks
+online version: https://docs.microsoft.com/en-us/powershell/module/az.databricks/new-azdatabricksvnetpeering
+schema: 2.0.0
+---
+
+# New-AzDatabricksVNetPeering
+
+## SYNOPSIS
+Creates vNet Peering for workspace.
+
+## SYNTAX
+
+```
+New-AzDatabricksVNetPeering -Name <String> -ResourceGroupName <String> -WorkspaceName <String>
+ [-SubscriptionId <String>] [-AllowForwardedTraffic] [-AllowGatewayTransit] [-AllowVirtualNetworkAccess]
+ [-DatabrickAddressSpaceAddressPrefix <String[]>] [-DatabrickVirtualNetworkId <String>]
```

Comment:
I missed these 2 parameter names in the design review. They should be named Databrick**s**AddressSpaceAddressPrefix and Databrick**s**VirtualNetworkId because "Databricks" is a special word